### DISCUSSION TITLE
服务报错。本地开发无法增加助手等内容

### DISCUSSION THREAD

**Discussion by anjiacm** [lobehub/lobehub#1188] [Development Question]
<img width="990" alt="image" src="https://github.com/lobehub/lobe-chat/assets/19515443/99afaea1-779e-4147-9a03-967c785a950d">
一旦出现图中的内容。lobechat就开始不对劲了

**Comment by arvinxx** [comment]
fetch failed 是请求出错，可能你要看看是不是本地网络问题

**Reply by anjiacm** [reply]
我是本地直接拉取源代码启动的。然后网络是直连的服务这样。

**Reply by arvinxx** [reply]
看报错是调用市场出的错。你看看能不能正常打开市场的索引文件？有可能是dns污染了

**Reply by anjiacm** [reply]
嗯 改了网络线路。目前看去正常

**Reply by cksdxz1007** [reply]
我也是本地部署，始终打不开market，即使用代理也不行
<img width="1235" alt="image" src="https://github.com/lobehub/lobe-chat/assets/21142070/1d18cde5-fc67-48bd-90c5-8e3cc5e6c786">

**Comment by hyrhyrh** [comment]
@anjiacm 您好，可以+ V  咨询几个问题吗

**Reply by anjiacm** [reply]
可以 微信号：qq-546167337
